bundle/manifests/sandboxed-containers-operator.clusterserviceversion.yaml

```diff
@@ -27,46 +32,72 @@ spec:
-    **Safe Multi-tenancy & Resource Sharing (CI/CD Jobs, CNFs, ..)**\n  If you are
-    providing a service to multiple tenants, it could mean that the\n  service workloads
-    are sharing the same resources (e.g., worker node). By\n  deploying in a dedicated
-    kernel, the impact of these workloads have on one\n  another is greatly reduced.\n\n-
-    **Additional Isolation with Native Kubernetes User Experience**\n  OpenShift sandboxed
-    containers is used as a compliant OCI runtime.\n  Therefore, many operational
-    patterns used with normal containers are still\n  preserved including but not
-    limited to image scanning, GitOps, Imagestreams,\n  and so on.\n\n# How to install\n
-    \ Read the information about the Operator and click Install.\n\n  On the Install
-    Operator page:\n\n  - Select preview-1.0 from the list of available Update Channel
-    options.\n  This ensures that you install the version of OpenShift sandboxed containers\n
-    \ that is compatible with your OpenShift Container Platform version.\n\n  - For
-    Installed Namespace, ensure that the Operator recommended namespace\n    option
-    is selected. This installs the Operator in the mandatory\n    openshift-sandboxed-containers-operator
-    namespace, which is automatically\n    created if it does not exist. Attempting
-    to install the OpenShift\n    sandboxed containers Operator in a namespace other
-    than\n    openshift-sandboxed-containers-operator causes the installation to fail.\n\n
-    \ - For Approval Strategy, ensure that Automatic, which is the default value,\n
-    \   is selected. OpenShift sandboxed containers automatically updates when a new\n
-    \   z-stream release is available.\n\n  - Click Install to make the Operator available
-    to the OpenShift sandboxed\n    containers namespace.\n\n  - The OpenShift sandboxed
-    containers Operator is now installed on your\n    cluster. You can trigger the
-    Operator by enabling the runtime on your cluster.\n    You can do this by creating
-    a KataConfig CRD instance. For this click\n    on \"create instance\" on the operator
-    overview page.\n\n# Documentation\nSee the official documentation [here](https://docs.openshift.com/container-platform/4.8/sandboxed_containers/understanding-sandboxed-containers.html). "
+  description: |-
+    OpenShift sandboxed containers based on the Kata Containers open source
+    project, provides an Open Container Initiative (OCI) compliant container
+    runtime using lightweight virtual machines, running your workloads in their own
+    isolated kernel and therefore contributing an additional layer of isolation
+    back to OpenShift's Defense-in-Depth strategy. For more information
+    [see](https://catalog.redhat.com/software/operators/detail/5ee0d499fdbe7cddc2c91cf5).
+
+    # Requirements
+    Your cluster must be installed on bare metal infrastructure with Red Hat Enterprise Linux CoreOS workers.
+
+    # Features & benefits
```

Comment:
I had not read this "features and benefits" before. Who wrote that? It's quite good ;-)

Reply:
Credit to @zanetworker afaik :-)